frontend/src/app/modules/challenges/online-editor/online-editor-page/online-editor-page.component.ts

```diff
@@ -52,29 +59,10 @@ export class OnlineEditorPageComponent implements OnDestroy, OnInit {
     }
 
     ngOnInit() {
-        const challengeId = this.activatedRoute.snapshot.params['id'];
-
         this.splitDirection = 'horizontal';
+        const challengeId = parseInt(this.activatedRoute.snapshot.params['id'], 10);
```

Comment:
I think it would be better to avoid magic number
```suggestion
   this.activatedRoute.paramMap.subscribe((params: ParamMap) => {
            const challengeId = +params.get('id')!;

            this.loadChallenge(challengeId);
        });
```

Reply:
i think it's not a magic number
it's a number system

the id parameter will not change during the user's work on the page. 
I think we can use any of these methods

but ok 😓